sphericart-torch/include/sphericart/torch.hpp

```diff
@@ -45,8 +46,8 @@ class SphericalHarmonics : public torch::CustomClassHolder {
     sphericart::SphericalHarmonics<float> calculator_float_;
 
     // CUDA implementation
-    sphericart::cuda::SphericalHarmonics<double> calculator_cuda_double_;
-    sphericart::cuda::SphericalHarmonics<float> calculator_cuda_float_;
+    sphericart::cuda::SphericalHarmonics<double>* calculator_cuda_double_ptr = nullptr;
```

Comment:
please use modern C++ here =) This should be `std::unique_ptr`!

Reply:
fixed in latest commit.